client/account-recovery/reset-password/reset-password-sms-form/index.jsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import FormTextInput from 'components/forms/form-text-input';
+import FormButton from 'components/forms/form-button';
+
+class ResetPasswordSmsForm extends Component {
+	render() {
+		const {
+			translate,
+		} = this.props;
+
+		return (
+			<Card>
+				<h2 className="reset-password-sms-form__title">
+					{ translate( 'Reset your password' ) }
+				</h2>
+				<p>
+					{ translate( 'Please enter the code you were sent by SMS. ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Please enter the code you were sent by SMS. It will look something like <code>63423423</code>. You may need to wait a few moments before it arrives.' )` *ES Score: 16*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).